src/main/java/com/debatetimer/domainrepository/organization/OrganizationDomainRepository.java

```diff
@@ -27,8 +27,8 @@ public List<Organization> findAll() {
                 .stream()
                 .collect(groupingBy(
                         OrganizationTemplateEntity::getOrganizationId,
-                        mapping(OrganizationTemplateEntity::toDomain, toList()))
-                );
+                        mapping(OrganizationTemplateEntity::toDomain, toList())
```

Comment:
정적 import 쓰지 않는 것이 컨벤션이었던 것 같아서 한번 체크 부탁할게요, 커찬!

```suggestion
                        Collectors.mapping(OrganizationTemplateEntity::toDomain, Collectors.toList())
```

Reply:
리뷰 반영 완료했습니다.